packages/react-components/cypress/stories/grid.spec.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react'
+import { composeStories, composeStory } from '@storybook/testing-react'
+import { SinonStub } from 'cypress/types/sinon'
+
+import * as stories from '../../stories/grid.stories'
+import { storiesCompositionToList } from '../utils/storybook'
+import {
+    checkGifSeen,
+    checkGifIsVisible,
+    checkGifKeyboardEvents,
+    checkGifMouseEvents,
+    GifTestUtilsContext,
+    resetGifEventsHistory,
+    setupGifTestUtils,
+} from '../utils/gif-test-utils'
+
+const GIFS_COUNT = 5
+
+const GridAPIError = composeStory(stories.GridAPIError, {})
+const composedStories = storiesCompositionToList(composeStories(stories)).filter(
+    (story) => story.key !== 'GridAPIError'
+)
+
+function getGridRoot() {
+    return cy.get('.giphy-grid')
```

Comment:
@pshoniuk there's a constant `Grid.className` that you could use here

Reply:
Great, I've updated it